sockeye/coverage.py

```diff
@@ -114,10 +118,10 @@ class GRUCoverage(Coverage):
     :param coverage_num_hidden: Number of hidden units for coverage vectors.
     """
 
-    def __init__(self, coverage_num_hidden: int, prefix='') -> None:
+    def __init__(self, coverage_num_hidden: int, prefix='cov_') -> None:
```

Comment:
I guess this is only backwards compatible because we don't set prefix, right? (just checking that this is backwards compatible.)